File: Ejercicios_if_php/practica_6.php
<?php
/*
EJERICICIO

6.Escriba un programa que pida un año y que
escriba si es bisiesto o no. Se recuerda
que los años bisiestos son multiplos de 4
pero los multiplos de 100 no los son, aunque
los multiplos  de 400 si
*/

// declaramos la variables

$año_actual=2020;
$coeficiente_100 = $año_actual/100;
$coeficiente_4 = $año_actual/4;


if (gettype($coeficiente_100)!= gettype($año_actual) and gettype($coeficiente_4)!= gettype($año_actual) ) {
    echo "No es años bisiesto";
}else{
    if ($año_actual % 4 == 0 or $año_actual % 400 ==0){
        echo "Es año bisiesto";
    }
}
   


?>

File: python/practica_6.py

# EJERCICIO

"""
6.Escriba un programa que pida un año y que
escriba si es bisiesto o no. Se recuerda
que los años bisiestos son multiplos de 4
pero los multiplos de 100 no los son, aunque
los multiplos  de 400 si
"""

#  SOLUCION

# declaracion de variables

año=int(input("Ingrese un año: "))

coeficiente_100= año/100
coeficiente_4 = año/4

print(type(coeficiente_100))
print(type(coeficiente_4))

print(type(año))

if  type(coeficiente_100)!= type(año) and  type(coeficiente_4)!= type(año) :
    print("No es año Bisiesto")
else:
    if año % 4 == 0 or año % 400 == 0:
        print("Es año bisiestos")
